src/core/Slider/Slider.tsx

```diff
@@ -284,28 +280,40 @@ export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
           pointerValue = roundValueToClosestStep(pointerValue, step, min);
           const [minVal, maxVal] = getAllowableThumbRange(activeThumbIndex);
           pointerValue = getBoundedValue(pointerValue, minVal, maxVal);
-          if (
-            !forceCallbackToRun &&
-            pointerValue === currentValues[activeThumbIndex]
-          ) {
+          if (pointerValue === currentValues[activeThumbIndex]) {
```

Comment:
`pointerValue === currentValues[activeThumbIndex] && 'onChange' === callbackType` but I am still thinking about the approach here...

Reply:
Not sure what you are asking for... I want to return for both callbackTypes, but I just want to fire callback if type is 'onChange'...
```
          if (pointerValue === currentValues[activeThumbIndex]) {
            if ('onChange' === callbackType) {
              onChange?.(currentValues);
            }
            return;
          }
```